test/lib.bash

```diff
@@ -208,7 +234,7 @@ function run_knative_serving_operator_tests {
 
   logger.info "Run tests of knative/serving-operator @ ${gitdesc}"
   env TEST_NAMESPACE='knative-serving' \
-    go test -v -tags=e2e -count=1 -timeout=30m -parallel=1 ./test/e2e \
+    go_test_e2e -tags=e2e -timeout=30m -parallel=1 ./test/e2e \
```

Comment:
Tests failed here. I was using `env TEST_NAMESPACE='knative-serving'` to override the namespace where tests create knative serving CR. But, it could not work with `go_test_e2e` which is bash function, not a command.

I think you need to export that variable, and after test restore it to original value (other places rely on the same name `TEST_NAMESPACE`).